### ISSUE TITLE
Sort searchable attributes

### ISSUE THREAD

**Issue by MarinPostma** [meilisearch/MeiliSearch#1005]
Like what was done in #943, it would be good if searchable attributes were also sorted in lexicographical order.

**Comment by hieetesh** [comment]
I will do it 👍

**Comment by Kerollmops** [comment]
The order of the searchable attributes is very important, it indicates the order of importance of the attributes in a document, we must not reorder them!

**Comment by MarinPostma** [comment]
@Kerollmops it doesn't matter anymore, since field pos and indexed_fields are two different things. The later is represented with a set anyways

**Comment by Kerollmops** [comment]
Yeah, but we must display the searchable attributes in the order the engine treat them.

**Comment by LegendreM** [comment]
@MarinPostma, As a user how can I change the attribute position for the criterion `attributes`? 🤔

**Comment by MarinPostma** [comment]
Can you?

**Comment by Kerollmops** [comment]
Updating a document to change the attributes order will not change the engines behavior as it will keep the global attributes order (the `searchable attibutes` setting), but updating the order of the `searchable attributes` setting must and will change the behavior of the `attributes` criterion.

**Comment by MarinPostma** [comment]
This needs to be tested and fixed, I will create an issue
